examples/javascript-developer/bin/build-executor.sh

```diff
@@ -6,8 +6,11 @@ rm -f .build/gptagent-executor.js
 mkdir -p .build
 npx esbuild executor.mjs --bundle --platform=node --loader:.node=file --outfile=.build/gptagent-executor.js
 
+rm -rf ./node_modules/@dqbd
+cp -a ../../node_modules/.pnpm/@dqbd+tiktoken@1.0.7/node_modules/@dqbd ./node_modules
```

Comment:
An alternative could be to run `npm install` for tiktoken it in the Docker container.